create callback/app.js

```diff
@@ -0,0 +1,8 @@
+var xhr = new XMLHttpRequest();
+
+xhr.onreadystatechange = function() {
+  
+};
+
+xhr.open('GET', 'sidebar.html');
```

Comment:
I think this should be
```suggestion
xhr.open('GET', 'index.html');
```
Since `sidebar.html` doesn't exist on this folder